hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/impl/TransferPrecompile.java

```diff
@@ -264,9 +290,46 @@ public void run(final MessageFrame frame) {
             }
         }
 
+        // track auto-creation child records if needed
+        if (autoCreationLogic != null) {
+            autoCreationLogic.submitRecords(
+                    infrastructureFactory.newRecordSubmissionsScopedTo(updater), false);
+        }
+
         transferLogic.doZeroSum(changes);
     }
 
+    private void replaceAliasWithId(
+            final BalanceChange change,
+            final List<BalanceChange> changes,
+            final MessageFrame frame,
+            final Map<ByteString, EntityNum> completedLazyCreates) {
+        final var receiverAlias = change.getNonEmptyAliasIfPresent();
+        validateTrueOrRevert(
+                !updater.aliases().isMirror(Address.wrap(Bytes.of(receiverAlias.toByteArray()))),
+                INVALID_ALIAS_KEY);
+        if (completedLazyCreates.containsKey(receiverAlias)) {
+            change.replaceNonEmptyAliasWith(completedLazyCreates.get(receiverAlias));
+        } else {
+            if (autoCreationLogic == null) {
+                autoCreationLogic = infrastructureFactory.newAutoCreationLogicScopedTo(updater);
+            }
+            final var lazyCreateResult =
+                    autoCreationLogic.create(change, ledgers.accounts(), changes);
+            validateTrue(lazyCreateResult.getLeft() == OK, lazyCreateResult.getLeft());
+            // charge lazy creation fees in gas
+            final var creationFeeInTinybars = lazyCreateResult.getRight();
+            final var creationFeeInGas = creationFeeInTinybars / frame.getGasPrice().toLong();
+            frame.decrementRemainingGas(creationFeeInGas);
```

Comment:
⚠️ Afaict this is double-charging for hollow creations. (Since the hollow creation costs are _also_ included in the `gasRequirement` ultimately charged [here](https://github.com/hashgraph/hedera-services/blob/2a0e4841be533a007d38f525fcb7fd8b22b77ef1/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/execution/HederaMessageCallProcessor.java#L112).

Reply:
🤦🏽‍♂️ Of course, removed this unneeded charging now